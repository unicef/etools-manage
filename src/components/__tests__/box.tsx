import 'react-testing-library/cleanup-after-each';
import React from 'react';
import { render } from 'react-testing-library';
import Box from '../box';

describe('Box', () => {
    test('renders', () => {
        render(<Box>Test</Box>);
    });

    test('mounts', () => {
        const { container } = render(<Box>Test</Box>);
        expect(container).toMatchSnapshot();
    });

});
