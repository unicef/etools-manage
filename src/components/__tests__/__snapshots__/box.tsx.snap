// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box mounts 1`] = `
<div>
  <div
    class="makeStyles-box-13"
  >
    Test
  </div>
</div>
`;
